tests/tasks/enable_epel.yml

```diff
@@ -20,6 +20,18 @@
       until: __epel_status is success
       retries: 6
       delay: 10
+      when: ansible_distribution_major_version != '7'
+
+    - name: Create EPEL-7
```

Comment:
```suggestion
    - name: Create EPEL-7
      # noqa command-instead-of-module
```

Reply:
I added it after `cmd` and before `register` the same way it is done in the task above.